versioneer.py

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Since versioneer is a third-party script, it should be ignored from any black/isort formatting.

Reply:
OK